src/assets/json/checklist/tests-web-checklist-en.json

```diff
@@ -0,0 +1,155 @@
+    "verifier": ["For each content, a mechanism is present to allow the user to pause it or hide it.", "This mechanism is accessible with mouse and keyboard navigation."], 
+    "resultat": ["User can pause or hide animations, movements, updates or blinks."], 
+    "exception": "", 
+	"raccourcis": "",
+	"moreInfo": "https://a11y-guidelines.orange.com/en/web/develop/common-navigation/#allowing-animations-control",
+	"profils": "",
+	"wcag": ["2.2.2"],
+	"ID": "testWebID-37",
+	"IDorigin": "testWebID-37"
+}, {
+	"themes": "Keyboard navigation",
+	"title": "Allow the use of the application with keyboard",
+	"type": ["Keyboard navigation"],
+	"tests": ["Scroll through the page with the keyboard using the <kbd>tab</kbd> or the <kbd>shift + tab</kbd> key.","Use all interactive elements (by pressing the <kbd>Enter</kbd> and <kbd>Space</kbd> keys for buttons/links, and the arrow keys for some components: a series of radio buttons, a tab system, etc.)."], 
+	"verifier": [["Everything that can be done with the mouse can be done using the keyboard and vice-versa"],["The focus movement order maintains a logical reading and usage flow."],["The focus does not get stuck on a part of the page (keyboard trap)."],["The focus indicator (outline) is visible on all interactive elements and has sufficient contrast."]],
+	"resultat": ["All interactive elements can be reached from a keyboard navigation", "All interactive elements are usable with keyboard interactions"],
+	"exception": "",
+	"raccourcis": "",
+	"moreInfo": "https://a11y-guidelines.orange.com/en/web/develop/keyboard-navigation/#allow-the-use-of-the-main-features-of-the-application-with-the-keyboard",
+	"profils": "",
+	"wcag": ["2.1.1"],
+	"ID": "testWebID-38",
+	"IDorigin": "testWebID-38"
+}, {
+	"themes": "Layout",
+	"title": "Use relative sizes and responsive design",
+	"type": ["Manual test"],
+	"tests": ["With Firefox, from the \"View\" menu, select \"Zoom\" and then \"Zoom Text Only\" and set the zoom level to 200%."],
+	"verifier": ["There is no truncated or hidden content and no unusable features"],
+	"resultat": ["No truncated or hidden contents and no unusable features"],
+	"exception": "",
+	"raccourcis": "",
+	"moreInfo": "https://a11y-guidelines.orange.com/en/web/develop/layout/#use-relative-size-to-allow-changing-text-size-and-responsive-design",
+	"profils": "",
+	"wcag": ["1.4.11"],
+	"ID": "testWebID-42",
+	"IDorigin": "testWebID-42"
+}, {
+	"themes": "Forms",
+	"title": "Make sure an accessible name is associated with each form field",
+	"type": ["Manual test - Code inspector"],
+	"tests": ["Run the code inspector of the browser on the \"Accessibility\" tab."],
+	"verifier": ["Each form field must have an accessible name."],
+	"result": ["Each field has at least one relevant accessible name and contains at least the text of the visible field label on the screen (a placeholder is not compliant)."],
```

Comment:
No idea why but it seems that this text does not appear on the page? (maybe double check the french version, it may have the same problem).

![screenshot-deploy-preview-603--a11y-guidelines_netlify_app-2024_12_23-11_44_47](https://github.com/user-attachments/assets/e815bee2-451b-42ca-bb3e-ff44c4b3a73a)